core/src/com/unciv/ui/trade/DiplomacyScreen.kt

```diff
@@ -137,11 +137,23 @@ class DiplomacyScreen(val viewingCiv:CivilizationInfo):CameraStageBaseScreen() {
             diplomacyTable.add(nextLevelString.toLabel()).row()
         }
 
+        val cultureBonus = if (viewingCiv.getEraNumber() in 0..1) 3f else if (viewingCiv.getEraNumber() in 2..3) 6f else 13f
+        val happinessBonus = if (viewingCiv.getEraNumber() in 0..1) "2" else "3"
+        val bonusMultiplier = if (viewingCiv.hasUnique("Food and Culture from Friendly City-States are increased by 50%")) 1.5f else 1f
+
+
         val friendBonusText = when (otherCiv.cityStateType) {
-            CityStateType.Cultured -> ("Provides [" + (3 * (viewingCiv.getEraNumber() + 1)).toString() + "] culture at 30 Influence").tr()
-            CityStateType.Maritime -> "Provides 3 food in capital and 1 food in other cities at 30 Influence".tr()
-            CityStateType.Mercantile -> "Provides 3 happiness at 30 Influence".tr()
-            CityStateType.Militaristic -> "Provides land units every 20 turns at 30 Influence".tr()
+            CityStateType.Cultured -> ("Provides [" + (cultureBonus * bonusMultiplier).toString().removeSuffix(".0") + "] culture when Friends").tr()
+            CityStateType.Maritime -> ("Provides [" + (2f * bonusMultiplier).toInt().toString().removeSuffix(".0") + "] food in capital when Friends").tr()
```

Comment:
In both cases, if you .toInt() it then you don't need to removeSuffix, since it's a whole number

Reply:
Not if you .toInt() it. (3.5).toInt().toString() = "3", not "3.0"
What I'm saying, is EITHER .toInt(), OR .removeSuffix(). There's no case where you'll need both.